pkgs/development/tools/wabt/default.nix

```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
+  patches = [ ./wasm-rt-impl.patch ];
```

Comment:
Does it make sense to send this upstream?

Reply:
https://github.com/WebAssembly/wabt/pull/1408 though it looks like it needs a bit of tweaking before it can be upstreamed

Reply:
Fixed patch.